src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/toggle.html.twig

```diff
@@ -22,20 +22,22 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-{% set id_property_name = column.options.toggle_field %}
+{% set id_property_name = column.options.primary_field %}
+{% set id_primary_key = record[id_property_name] %}
 {% set record_id = record[id_property_name] %}
 
 {% set valid = (record[column.id] is same as('1')) %}
 
-<i
+<div class="text-center">
+  <i
     class="material-icons ps-togglable-row grid-toggler-icon-{% if valid == true %}valid{% else %}not-valid{% endif %}"
-    data-toggle-field-name="{{ column.options.field }}"
-    data-toggle-field-id="{{ record_id }}"
-    data-toggle-url="{{ column.options.url }}"
->
-{% if valid == true %}
-    check
-{% else %}
-    clear
-{% endif %}
-</i>
+    data-toggle-url="{{ path(column.options.route, {(column.options.route_param_id) : id_primary_key})}}"
+  >
+    {% if valid == true %}
```

Comment:
`{% if isValid %} is enough here as valid is a boolean by design: rename the variable :)